docs/reference/configuration.rst

```diff
@@ -182,12 +182,12 @@ Full Configuration Options
                     - bundles/sonataadmin/dist/bootstrap/bootstrap.min.css
                     - bundles/sonataadmin/dist/components-font-awesome/css/font-awesome.min.css
                     - bundles/sonataadmin/dist/ionicons/css/ionicons.min.css
-                    - bundles/sonataadmin/dist/admin-lte/css/AdminLTE.min.css
+                    - bundles/sonataadmin/dist/admin-lte/css/AdminLTE-without-plugins.min.css
```

Comment:
Default admin-lte has custom styles for select2, fullcalendar and bootstrap-social.

From those, we are only using select2, and the 4 version has imcompatible styles for it: https://github.com/ColorlibHQ/AdminLTE/issues/668